mergify_engine/branch_updater.py

```diff
@@ -259,4 +291,6 @@ def update_with_git(ctxt, method="merge", user=None):
             "Rebasing a branch for a forked private repository is not supported by GitHub"
         )
 
-    raise AuthenticationFailure("No valid OAuth tokens")
+    raise AuthenticationFailure(
+        f"No registered token allows Mergify to push to `{ctxt.pull['head']['label']}`"
```

Comment:
```suggestion
        f"No registered tokens allow Mergify to push to `{ctxt.pull['head']['label']}`"
```
We should use plural form here since we're talking about the registered tokens.